src/com/dakl/Main.java

```diff
@@ -16,9 +16,12 @@ public static void main(String[] args) {
 
         CheckerBoard game = CheckerBoard.blankBoard();
 
+        // REVIEW : Setup method
         Space start_loc = game.getBoardSpace(Main.parseSpace("C4R5"));
         Team r = game.red;
+        // REVIEW : calling static methods
         Main.threeByThree(game.black, new Team[] {r, r, r, r}, start_loc, true, game);
```

Comment:
When you're calling a static method in the same class, you don't have to use the class name - e.g. you could just call threeByThree(...) here (instead of Main.threeByThree)